test/input/closeness_centrality_unweighted.gt

```diff
@@ -1,9 +1,8 @@
 element Vertex end
 element Edge end
 
-const edges : edgeset{Edge}(Vertex, Vertex) = load ("../../test/graphs/test.el");
+const edges : edgeset{Edge}(Vertex, Vertex) = load ("../../../../baghdadi/data3/socLive/socLive_gapbs.sg");
```

Comment:
Change the paths back to test.el? 
Please be more careful before you commit code. Make sure you do "git diff" to double check that you don't commit temporary changes.